ogre/src/OgreRenderEngine.cc

```diff
@@ -78,7 +78,11 @@ RenderEngine *OgreRenderEnginePlugin::Engine() const
 OgreRenderEngine::OgreRenderEngine() :
   dataPtr(new OgreRenderEnginePrivate)
 {
-  this->ogrePaths.push_back(std::string(OGRE_RESOURCE_PATH));
+  const char *env = std::getenv("OGRE_RESOURCE_PATH");
+  std::string ogre_resource_path = (env) ? std::string(env) :
+      OGRE_RESOURCE_PATH;
```

Comment:
How about still adding the compile-time resource path even if there's an environment variable? This way the same application could use both pools of resources.

Also, a style nitpick, we use `camelCase` for variables.

Reply:
Done.